### PR TITLE
Apply wemake-python-styleguide

### PR DESCRIPTION
**Pull request by AlwxSin**
#15 drop python2 support
#16 add wemake-python-styleguide as dependency and refactor
remove tox file, use travis instead
update readme and docs
use full power of poetry (ref #8)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 install:
-  - pip install tox-travis wheel pypandoc
+  - poetry install
 
 script:
-  - tox
+    - poetry run flake8 dump-env tests docs
+    - poetry run pytest
+    - poetry check
+    - poetry run pip check
```
Comment (sobolevn) [inline .travis.yml]: safety?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,23 @@ This command will:
 7. dump all the keys and values into the `.env` file
 
 
+## Advanced Usage
+```bash
+$ dump-env -t .env.template -p 'SECRET_ENV_' -p 'ANOTHER_SECRET_ENV_' > .env
+```
+
+This command will do the same as in quickstart demo but variables with second prefix will replace variables with first prefix.
```
Comment (sobolevn) [inline README.md]: Are you sure about this wording?

Comment (sobolevn) [reply README.md]: Maybe it is better to rephrase it to:
```
## Multiple prefixes

.. example

This command will do pretty much the same thing as with one prefix. But, it will replace multiple prefixes.
```

```bash
$ export SECRET_TOKEN='very secret string'
$ export ANOTHER_SECRET_ENV_VALUE='0'
$ export SECRET_ANSWER='13'
$ export ANOTHER_SECRET_ENV_ANSWER='42'
$ dump-env -p SECRET_ -p ANOTHER_SECRET_ENV_
ANSWER=42
TOKEN=very secret string
VALUE=0
```

What order should user expect? Who would win? `SECRET_` value or `ANOTHER_SECRET_ENV_` value?

Comment (AlwxSin) [reply README.md]: Makes sense. I'll fix that.

File: .travis.yml (hunk 1 above)
Comment (AlwxSin) [reply .travis.yml]: Done

File: README.md (hunk 2 above)
Comment (AlwxSin) [reply README.md]: @sobolevn how about now?

**Comment by sobolevn** [comment]
AWESOME! 🎉
